js/editor-libs/mce-utils.js

```diff
@@ -26,16 +26,25 @@ module.exports = {
     isPropertySupported: function(dataset) {
         'use strict';
 
-        var property = dataset['property'];
         /* If there are no 'property' attributes,
            there is nothing to test, so return true. */
-        if (property === undefined) {
+        if (dataset['property'] === undefined) {
```

Comment:
Any specific reason you do not want to store this in a variable?

Reply:
Because then I'd have extra temporary state - I only need this variable so I can immediately slice it. So I think this:

```
var propertiesAttribute = dataset['property'];
if (propertiesAttribute === undefined) {
            return true;
        }

/* 'property' may be a space-separated list of properties. */
var properties = propertiesAttribute.split(' ');
```

is less clear than this:

```
if (dataset['property'] === undefined) {
    return true;
}

/* 'property' may be a space-separated list of properties. */
var properties = dataset['property'].split(' ');
```

But it's not my code, I'd be happy to change it of you prefer.